modules/core/ui/context/config/optional/block.block.farm_context.yml

```diff
@@ -0,0 +1,23 @@
+langcode: en
+status: true
+dependencies:
```

Comment:
Needs an enforced dependency so that it gets removed when `farm_ui_context` is uninstalled.